tests/Parse/UnderscorePatternSpec.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Parse.UnderscorePatternSpec where
+
+import AST.Source qualified as Src
+import Data.ByteString qualified as BS
+import Helpers.Instances ()
+import Parse.Pattern qualified as Pattern
+import Parse.Primitives qualified as P
+import Reporting.Annotation qualified as A
+import Test.Hspec ( Spec ) 
+import Test.Hspec qualified as Hspec 
+
+data ParseError
+  = ExprError P.Row P.Col
+  | OtherError String P.Row P.Col
+  deriving (Show, Eq)
+
+spec :: Spec
+spec = do
+  Hspec.describe "Wildcard patterns" $ do
+    Hspec.it "regression test" $
```

Comment:
Would you mind importing `describe` and `it` unqualified, as well as adding a line of whitespace between each test?

Just so the formatting is the same between all tests.